msg.go

```diff
@@ -921,7 +921,7 @@ func packStructValue(val reflect.Value, msg []byte, off int, compression map[str
 				copy(msg[off:off+len(s)], s)
 				off += len(s)
 			case `dns:"octet"`:
-				bytesTmp := make([]byte, 0)
+				bytesTmp := make([]byte, 256*4+1)
```

Comment:
I'm unclear what 256*4+1 is.  I see it's used more than once in this file.  Should it be a constant?